org.jacoco.doc/docroot/doc/build.html

```diff
@@ -267,18 +267,22 @@ <h2>Compilation and testing with different JDKs</h2>
   <li><code>mvn clean verify -Djdk.version=6 -Dbytecode.version=6</code></li>
   <li><code>mvn clean verify -Djdk.version=7 -Dbytecode.version=7</code></li>
   <li><code>mvn clean verify -Djdk.version=8 -Dbytecode.version=8</code></li>
-  <li><code>mvn clean verify -Djdk.version=8 -Dbytecode.version=8 -Decj</code></li>
+  <li><code>mvn clean verify -Djdk.version=8 -Dbytecode.version=8</code></li>
   <li><code>mvn clean verify -Djdk.version=9 -Dbytecode.version=9</code></li>
   <li><code>mvn clean verify -Djdk.version=10 -Dbytecode.version=10</code></li>
-  <li><code>mvn clean verify -Djdk.version=11 -Dbytecode.version=11</code></li>
+  <li><code>mvn clean verify -Djdk.version=11 -Dbytecode.version=11 -Decj</code></li>
```

Comment:
While Java 11 should now be listed twice: With and without `-Decj`

Reply:
Done